test/configuration/configuration-helper.js

```diff
@@ -0,0 +1,19 @@
+const { CIRCLECI } = process.env;
+const video = require('wdio-video-reporter');
+const fileName = CIRCLECI ? 'circleci' : 'local';
```

Comment:
Probably best to have your imports at the top, and we don't seem to be using the variable anywhere else so it's not necessary.
```suggestion
const video = require('wdio-video-reporter');

const fileName = process.env.CIRCLECI ? 'circleci' : 'local';
```